### DISCUSSION TITLE
how-to/

### DISCUSSION THREAD

**Discussion by giscus** [aidenlx/obsidian-zotero#83] [Announcements]
# how-to/

WIP

https://obzt.aidenlx.top/how-to/

**Comment by jimwillshire** [comment]
Hi,

This looks great and I would love to get started with it, I've followed your installation guide and it looks to be correct, but I'm not sure how to get started!!

Could you please provide some tips to get started?

Thanks


Jim

**Comment by Qtabb** [comment]
Hello I am new here. I use Zotero mobile and Obsidian on Ipad pro. Is this plugin support my device. Thank you.
